src/App.tsx

```diff
@@ -1,61 +1,33 @@
-import React,   { useEffect, useState } from 'react';
-import { createClient } from "@supabase/supabase-js";
+import React from 'react';
+import { createClient } from '@supabase/supabase-js';
 import './App.css';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import Login from './pages/Login';
+import SignUp from './pages/Signup';
+import Home from './pages/Home';
 
-
-const supabase = createClient(process.env.REACT_APP_PROJECT_URL || '', process.env.REACT_APP_API_KEY || '');
-
-interface Country {
-  id: number;
-  name: string;
-}
-
-type CountryList = Country[];
-
-interface CountryName {
-    name: string
-}
+export const supabase = createClient(
```

Comment:
is this being used on this file? If not we can move it to it's own file if it being exported to other areas in the app.